app/views/curation_concerns/base/_metadata.html.erb

```diff
@@ -1,12 +1,18 @@
 <h2><%= t('.header') %></h2>
-<table class="table table-striped <%= dom_class(presenter) %> attributes" <%= presenter.microdata_type_to_html %>>
-  <thead>
-    <tr><th>Attribute Name</th><th>Values</th></tr>
-  </thead>
-  <tbody>
-    <%= render 'attribute_rows', presenter: presenter %>
-    <%= presenter.attribute_to_html(:embargo_release_date) %>
-    <%= presenter.attribute_to_html(:lease_expiration_date) %>
-    <%= presenter.attribute_to_html(:rights) %>
-  </tbody>
-</table>
+<% attributes ||= [ :embargo_release_date, :lease_expiration_date, :rights] %>
+<% presentations = attributes.map { |attribute| presenter.attribute_to_html(attribute) }.reject { |x| x.empty? } %>
```

Comment:
I think it would be great to put this logic in the presenter.  So we can say `presenter.metadata_empty?`